README.md

```diff
@@ -56,7 +56,7 @@ end
 
 ##
 # [system] keep the answer concise
-# [user] [{:type=>:image_url, :image_url=>{:url=>"https://upload.wikimedia.org/wikipedia/commons/b/be/Red_eyed_tree_frog_edit2.jpg"}}]
+# [user] https://upload.wikimedia.org/wikipedia/commons/b/be/Red_eyed_tree_frog_edit2.jpg
```

Comment:
Maybe we should specify whether this was sent as a URI/File or just text?

Reply:
Not in scope for this PR. But agreed, we should handle more than "text" and that goes for the response level too.